gooddata-dbt/gooddata_dbt/gooddata/config.py

```diff
@@ -47,11 +47,17 @@ class GoodDataConfigOrganization(Base):
     data_product_ids: List[str] = attr.field(default=list)
 
 
+@attrs.define(auto_attribs=True, kw_only=True)
+class GoodDataGlobalConfig(Base):
+    test_insights_parallelism: Optional[int] = 1
```

Comment:
What does this attribute represent? Is it the number of threads?

Reply:
yes, it represents the number of threads which are opened and serve for the execution of all insights.